src/aics-image-viewer/shared/utils/controlPointsToLut.ts

```diff
@@ -46,5 +48,5 @@ export function controlPointsToLut(controlPoints) {
   ctx.fillStyle = grd;
   ctx.fillRect(0, 0, 256, 1);
   const imgData = ctx.getImageData(0, 0, 256, 1);
-  return imgData.data;
+  return new Uint8Array(imgData.data.buffer);
```

Comment:
my only concern here is that for functions that get called a LOT, there can be some performance degradation by creating lots of objects if you don't have to. garbage collection etc..

Reply:
This fixed a type conflict - `CanvasRenderingContext2D.getImageData` returns a `Uint8ClampedArray`, which TS evaluates as incompatible with the plain unclamped version. (I actually added the constructor at the site you commented about above in a previous PR, then realized it could be moved upstream in this one.) It might be possible to re-type this value downstream, e.g. turn `Channel.lut` into a `Uint8Array | Uint8ClampedArray`, but since this copies no data (as you mentioned above, though it's not obvious given the syntax) it's not the worst fix ever.